pytest_rpc/__init__.py

```diff
@@ -32,7 +32,25 @@
 
 
 # ======================================================================================================================
-# Functions
+# Functions: Private
+# ======================================================================================================================
+def _capture_marks(items, marks):
+    """Add XML properties group to each 'testcase' element that captures the specified marks.
+
+    Args:
+        items (list(_pytest.nodes.Item)): List of item objects.
+        marks (list(str)): A list of marks to capture and record in JUnitXML for each 'testcase'.
+    """
+
+    for item in items:
+        for mark in marks:
+            marker = item.get_marker(mark)
+            if marker is not None:
+                item.user_properties.append((mark, marker.args[0]))
```

Comment:
What will this do for marks that have different signatures
`@pytest.mark.jira('ASC-123')`
`@pytest.mark.jira('ASC-123', 'ASC-456')`
`@pytest.mark.skip(reason='because I said so')`

Reply:
Ok, I see that you are testing for the second case and the third is one that is out of scope for this PR.